src/Core/Services/DeviceTrustCryptoService.cs

```diff
@@ -99,5 +99,33 @@ public async Task<DeviceResponse> TrustDeviceIfNeededAsync()
             await SetShouldTrustDeviceAsync(false);
             return response;
         }
+
+        public async Task<bool> IsDeviceTrustedAsync()
+        {
+            var existingDeviceKey = await GetDeviceKeyAsync();
+            return existingDeviceKey != null;
+        }
+
+        public async Task<UserKey> DecryptUserKeyWithDeviceKeyAsync(string encryptedDevicePrivateKey, string encryptedUserKey)
+        {
+            // Get device key
```

Comment:
⛏️ Remove comment, it's unnecessary given that one can read it in the next line method.
```suggestion

```